en/08_Changelogs/6.0.0.md

```diff
@@ -746,23 +767,14 @@ class MyCustomPage extends Page
 > - Reverse the condition - if you are currently adding a field if some condition is true, change your logic so you instead remove the scaffolded form field if that condition is false.
 > - Use the `scaffold_cms_fields_settings` configuration to ensure those form fields are not scaffolded. You can then leave your logic as is for those fields.
 
-See the [scaffolding](/developer_guides/model/scaffolding/#scaffolding-options) section for more details about using these options.
-
-As part of your CMS 6 upgrade, you should check all of the page types in your project and in any modules you maintain to ensure the correct form fields are available in the appropriate tabs. You should also check [`Extension`](api:SilverStripe\Core\Extension) subclasses that you know get applied to pages to ensure fields aren't being scaffolded from those that you want to keep hidden.
+As part of your CMS 6 upgrade, you should check all of the page types in your project and in any modules you maintain to ensure the correct form fields are available in the appropriate tabs. You should also check [`Extension`](api:SilverStripe\Core\Extension) subclasses that you know get applied to pages to ensure fields aren't being scaffolded from those that you want to keep hidden. As with all configuration, you can set `scaffold_cms_fields_settings` configuration in your `Extension` subclass and it will be merged into the configuration for the classes it gets applied to.
 
 #### What if I don't have time to upgrade all of my page types?
 
 If you have a lot of complex page types and extensions, upgrading all of them to account for the new scaffolding might be a large task. If you want to avoid upgrading your `getCMSFields()` and `updateCMSFields()` implementations initially, you can use the `restrictRelations` and `restrictFields` scaffolding options in the `scaffold_cms_fields_settings` configuration property for your pages. You can then declare that only the fields introduced in parent classes should be scaffolded.
 
 The below YAML configuration can be used as a base for this workaround. It will work for all page types available in commercially supported modules. If you use page types provided in third-party modules, you may need to add configuration for those as well.
 
-> [!WARNING]
-> Note that this is explicitly intended as a temporary workaround, so that you can focus on other areas of the upgrade first, and come back to your page form fields later.
->
-> As more community modules are upgraded to account for form field scaffolding in their page types and extension classes, you may
-> need to add more fields to this list. To avoid having to continuously update these lists it's recommended that you take the time
-> to update your `getCMSFields()` and `updateCMSFields()` implementations as soon as you have time to do so.
```

Comment:
Moved to be below the collapsed section because otherwise the collapsed element is easy to miss